pcsx2-qt/MainWindow.cpp

```diff
@@ -928,7 +932,7 @@ void MainWindow::restoreStateFromConfig()
 	}
 }
 
-void MainWindow::updateEmulationActions(bool starting, bool running, bool stopping)
+void MainWindow::updateEmulationActions(bool starting, bool running, bool stopping, bool hardcore_mode)
```

Comment:
Is there a benefit to having an additional argument here? Or would having a `const bool hardcore_mode_active = Achievements::IsHardcoreModeActive()` at the top of this function work?
It looks like IsHardcoreModeActive() should be safe to call in any VM state.